spec/factories/hotels.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :hotel do
+    hotel_place_id { "MyString" }
+    name { "MyString" }
+    email { "MyString" }
+    phone { "MyString" }
+    website { "MyString" }
+    address { "MyString" }
+    image_url { "MyString" }
+    wheelchair { false }
+    stars { 1 }
```

Comment:
I suggest using a combination of Faker, [SecureRandom.hex](https://ruby-doc.org/stdlib-3.0.2/libdoc/securerandom/rdoc/SecureRandom.html#module-SecureRandom-label-Examples) and [rand](https://ruby-doc.org/3.2/Kernel.html#method-i-rand) to fill these in.

Reply:
Added faker data 👍